packaging/setup/ovirt_engine_setup/engine/constants.py

```diff
@@ -266,12 +250,6 @@ class FileLocations(object):
         'runner.wsgi',
     )
```

Comment:
HTTPD_RUNNER_WSGI_SCRIPT is also not needed, right?

Reply:
Oh yeah, missed that one.